package.json

```diff
@@ -52,6 +56,7 @@
     "eslint_d": "11.1.1",
     "husky": "7.0.4",
     "lint-staged": "11.1.2",
+    "parcel": "^2.4.0",
```

Comment:
adding parcel as a second javascript compiler when there already is a webpack setup seems a bit overkill, could use webpack dev server --watch script to run the demo page instead?

Reply:
(to get the fairplay fix merged quickly I'd suggest opening a separate PR with just the fairplay fix, the demo page could then be a separate `feat:` commit for the automated release notes)